wiki/Tournaments/Official_support/en.md

```diff
@@ -77,7 +77,7 @@ In addition, all promotional material or any services associated with a tourname
 **The tournament must also abide by the following practices throughout:**
 
 - All registrants must be screened by the account support team before play commences.[^play-commence] See [tournament screening](#tournament-screening) for more details.
-- If a user satisfies the sign-up criteria (if any), the tournament must not prevent those who pass the screening from participating without both ample evidence presented publicly against them and the approval of the account support team.
+- If a user satisfies the sign-up criteria (if any) and is not [filtered](#registrant-filtering-seeding-and-automated-refereeing) out through other means, the tournament must not prevent those who pass the screening from participating without both ample evidence presented publicly against them and the approval of the account support team.
```

Comment:
only this section is relevant and not the umbrella section (that i suggested to be removed)

```suggestion
- If a user satisfies the sign-up criteria (if any) and is not [filtered](#registrant-filtering-and-seeding) out through other means, the tournament must not prevent those who pass the screening from participating without both ample evidence presented publicly against them and the approval of the account support team.
```